tensorflow/python/ops/nn_grad.py

```diff
@@ -82,6 +82,36 @@ def _Conv3DGrad(op, grad):
                                            padding=op.get_attr("padding"))]
 
 
+@ops.RegisterGradient("Conv3DBackpropInputV2")
+def _Conv3DBackpropInputGrad(op, grad):
+  return [None,
+      nn_ops.conv3d_backprop_filter_v2(grad,
```

Comment:
Hi, 


I've been looking  for some useful stuff and found an interesting site with  useful advices, please take a look <http://agree.pumpjuice.com/e4ann>

ziney2y